indico/modules/events/contributions/models/contributions.py

```diff
@@ -47,8 +47,25 @@ def _get_next_friendly_id(context):
     return increment_and_get(Event._last_friendly_contribution_id, Event.id == event_id)
 
 
+class CustomFieldsMixin(object):
+    """Methods to process custom field data."""
+
+    def set_custom_field(self, field_id, field_value):
+        fv = self.get_field_value(field_id, raw=True)
+        if not fv:
+            field_value_cls = type(self).field_values.prop.mapper.class_
+            fv = field_value_cls(contribution_field=self.event_new.get_contribution_field(field_id), data=field_value)
```

Comment:
But the `field_value` is the actual data value, if I don't set it at `data= field_value`, then where is it gonna go?!